packages/design-preview/src/components/index.tsx

```diff
@@ -19,40 +26,68 @@ interface PreviewProps {
 	onClickCategory?: ( category: Category ) => void;
 	actionButtons: React.ReactNode;
 	recordDeviceClick: ( device: string ) => void;
+	siteId: number;
+	stylesheet: string;
+	selectedFontVariation: GlobalStylesObject | null;
+	onSelectFontVariation: ( variation: GlobalStylesObject | null ) => void;
+	onGlobalStylesChange: ( globalStyles: GlobalStylesObject | null ) => void;
 }
 
 const INJECTED_CSS = `body{ transition: background-color 0.2s linear, color 0.2s linear; };`;
 
 const getVariationBySlug = ( variations: StyleVariation[], slug: string ) =>
 	variations.find( ( variation ) => variation.slug === slug );
 
-const Preview: React.FC< PreviewProps > = ( {
+// @todo Get the style variations of theme, and then combine the selected one with colors & fonts for consistency
+const Preview: React.FC< DesignPreviewProps > = ( {
 	previewUrl,
 	title,
 	author,
 	categories = [],
 	description,
 	shortDescription,
 	pricingBadge,
-	variations = [],
+	variations,
 	selectedVariation,
```

Comment:
Nit: as we now have style and font variation, maybe rename this one to `selectedStyleVariation`? Or will than explode the diff of this PR? 😄

Reply:
I'll rename it in the follow-up PRs 👍

Reply:
Sounds good!